backend/program/scrapers/jackett.py

```diff
@@ -0,0 +1,99 @@
+""" Jackett scraper module """
+from datetime import datetime
+from typing import Optional
+from pydantic import BaseModel
+from utils.logger import logger
+from utils.request import get
+from utils.settings import settings_manager
+from utils.utils import parser
+from requests.exceptions import ReadTimeout
+import xmltodict
+
+class JackettConfig(BaseModel):
+    url: Optional[str]
+    api_key: Optional[str]
+
+
+class Jackett:
+    """Scraper for Jackett"""
+
+    def __init__(self):
+        self.settings = "jackett"
+        self.class_settings = JackettConfig(**settings_manager.get(self.settings))
+        self.last_scrape = 0
+        self.initialized = False
+
+        if self.validate_settings():
+            self.initialized = True
+
+    def validate_settings(self) -> bool:
+        """Validate the Jackett class_settings."""
+        if self.class_settings.api_key and self.class_settings.url:

```

Comment:
Needs improvement

Reply:
Jackett doesn't have like a `/user` endpoint, should I just use ping from our request module?

Reply:
improved. It now does a test query with jackett